src/helpers.js

```diff
@@ -11,7 +11,13 @@ export const mapState = normalizeNamespace((namespace, states) => {
       let state = this.$store.state
       let getters = this.$store.getters
       if (namespace) {
-        const module = getModuleByNamespace(this.$store, 'mapState', namespace)
+        if (typeof namespace === 'function') {
+          namespace = namespace.call(this, this)
```

Comment:
Because the component may be loaded multiple times, the namespace of the subsequent component is always the result of the first execution, so the code cannot override namespace = namespace.call(this, this), we can use the local variable let ns = namespace.call(this, this)